src/aap_eda/services/activation/manager.py

```diff
@@ -76,7 +76,58 @@ def _run_with_lock(self: HasDbInstance, *args, **kwargs):
     return _run_with_lock
 
 
-class ActivationManager:
+class StatusManager:
```

Comment:
@Alex-Izquierdo Can we move this to a separate file? This file is already too big. It also allows us to carve out other pieces into smaller classes like the Restart Policies

Reply:
Hi @mkanoor I've created status_manager file and renamed manager to activation_manager. Let me know if that's fine to you.